community-operators/microcks/microcks-operator.package.yaml

```diff
@@ -0,0 +1,4 @@
+packageName: microcks
+channels:
+- name: alpha
+  currentCSV: microks-operator.v0.1.0
```

Comment:
nit: newline

Reply:
Also this is misspelled. Please make sure this CSV name matches the `metadata.name` field in the CSV.

Reply:
Typo is fixed in new commit